### PR TITLE
CR#2

### PR DESCRIPTION
**Pull request by thcdevcl**
FIXED CR#1

### CODE REVIEW

File: assets/styles.css [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+  background-color: #f72210 !important;
+  height: 95px;
+}
+
+.opinion-description {
+  color: #666;
+  font-size: 14px;
+  line-height: 21px;
+  text-transform: uppercase;
+  font-size: 13px/18px;
```
Comment (mcrd25) [inline assets/styles.css:192]: Is this supposed to be `font`? if so then you don't need font-size and line-height in previous lines in this class, but yes please change to `font`.

File: assets/styles.css [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+  border-top: 1px solid lightgray;
+  border-bottom: 1px solid lightgray;
+  padding: 16px 0;
+}
+
+.section-pick-category {
+  color: #f72210;
+  font-size: 20px;
+  line-height: 20px;
+  font-size: 16px/26px;
```
Comment (mcrd25) [inline assets/styles.css:290]: Should be `font` :). Also remove lines 288 and 289 is you are using this property as they are not necessary.

**Review by mcrd25 — APPROVED** [review]
# Yay Yay Yay, Project Looks great, you can now 🚢  it to master and 🚀 . WooHoo!
Thank you for implementing changes requested, project really well done! I'm proud!

**One thing though, please address review comments before merging.**

Really well done! Looks great 😄 💯 🥇

**Comment by thcdevcl** [comment]
Thanks for your time! Will fix those fonts and will merge then!

**Pull request merged**